src/engine/ExecuteUpdate.h

```diff
@@ -10,6 +10,14 @@
 #include "parser/ParsedQuery.h"
 #include "util/CancellationHandle.h"
 
+// Metadata about the time spent on different parts of an update.
+struct UpdateMetadata {
+  std::chrono::milliseconds triplePreparationTime_ =
+      std::chrono::milliseconds::zero();
+  std::chrono::milliseconds insertionTime_ = std::chrono::milliseconds::zero();
+  std::chrono::milliseconds deletionTime_ = std::chrono::milliseconds::zero();
```

Comment:
If you use an alias for the type, and a constant for zero, the you only gain one line, and then thing gets more readable.